pkg/webhook/webhook_roles.go

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package webhook
+
+import (
+	"context"
+	v1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func CreateWebhookRole(client crclient.Client,
+	ctx context.Context,
+	saName string) error {
+
+	clusterRole, err := getSpecRoleBinding(saName)
+	if err != nil {
+		return err
+	}
+
+	if err := client.Create(ctx, clusterRole); err != nil {
+		if !apierrors.IsAlreadyExists(err) {
+			return err
+		}
+		existingCfg, err := getClusterRole(ctx, saName, client)
+		if err != nil {
+			return err
+		}
+		clusterRole.ResourceVersion = existingCfg.ResourceVersion
+		err = client.Update(ctx, clusterRole)
+		if err != nil {
+			return err
+		}
+		log.Info("Updated webhook server cluster role")
+	} else {
+		log.Info("Created webhook server cluster role")
+	}
+
+	return nil
+}
+
+func getSpecRoleBinding(saName string) (*v1.ClusterRole, error) {
+
+	labels := map[string]string{
+		"app.kubernetes.io/name":    saName,
```

Comment:
I would not use here service account name

Reply:
Is addressed in https://github.com/devfile/devworkspace-operator/pull/126/commits/681ffb7526cd805834b9baf2543caede45322266